config/samples/apim_v1alpha1_ratelimitpolicy.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   namespace: kuadrant-system
   labels:
     app: toystore
+  annotations:
```

Comment:
I think this should be part of the spec as per our discussion today 

```
targetRef:
   type: VirtualService
   name: myservice
```

Reply:
Updated. Just instead of `targetRef`, I have chosen `networkingRef`